pkg/activator/util/context.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This file contains the context manipulation routines used within
+the activator binary.
+*/
+
+package util
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/types"
+	"knative.dev/serving/pkg/apis/serving/v1alpha1"
+)
+
+type revisionKey struct{}
+type revIDKey struct{}
```

Comment:
Can we use like this
```
type (
	revisionKey struct{}
        revIDKey struct{}
)
```

Reply:
From language perspective: yes we can.
Corollary is that I have never seen this be done anywhere :)

Reply:
Oh actually I have referred from [here](https://golang.org/ref/spec#Type_declarations) :smile: